meta/SaiSerialize.cpp

```diff
@@ -14,6 +14,14 @@
 
 using json = nlohmann::json;
 
+const std::unordered_map<sai_link_event_damping_algorithm_t, std::string> sai_link_event_damping_algorithm_to_name_map = {
```

Comment:
i think sai_link_event_damping_algorithm_t should be renamed to sai_redis_link_event_damping_algorithm_t

Reply:
since this is not a part of SAI enums

Reply:
done.